frontend/gwms_renew_proxies.py

```diff
@@ -88,6 +92,8 @@ def __init__(self, vo, fqan):
         self.name = vo
         if fqan.startswith('/%s/Role=' % vo):
             pass
+        elif fqan.startswith('/%s/' % vo):
```

Comment:
Instead of adding an `elif` here, let's just loosen the restriction in the initial if to:

```
if fqan.startswith('/%s/' % vo):
```